marshmallow_sqlalchemy/fields.py

```diff
@@ -3,30 +3,28 @@
 from marshmallow import fields
 
 
-def get_primary_column(model):
-    """Get primary key column for a SQLAlchemy model.
+def get_primary_columns(model):
+    """Get primary key columns for a SQLAlchemy model.
 
     :param model: SQLAlchemy model class
-    :raise: RuntimeError if model has multiple primary key columns
     """
-    columns = model.__mapper__.primary_key
-    if len(columns) == 1:
-        return columns[0]
-    raise RuntimeError('Model {0!r} has multiple primary keys'.format(model))
+    return model.__mapper__.primary_key
 
+def ensure_list(value):
+    return value if isinstance(value, list) else [value]
```

Comment:
It might make sense to use `marshmallow.utils.is_iterable_but_not_string` here so that we can deserialize other collection types.